pepper-apis/src/main/java/org/broadinstitute/ddp/service/PdfGenerationService.java

```diff
@@ -138,16 +140,35 @@ public InputStream generateFlattenedPdfForConfiguration(PdfConfiguration configu
                                        Handle handle) throws IOException {
         Participant participant = loadParticipantData(handle, configuration, userGuid);
         Map<Long, ActivityResponse> instances = loadActivityInstanceData(handle, configuration, participant);
+        Long languageId = null;
+        if (participant.getUser().getProfile() != null) {
+            languageId = participant.getUser().getProfile().getPreferredLangId();
+        }
+        if (languageId == null) {
+            //use study default language if exists
+            StudyLanguageDao studyLanguageDao = handle.attach(StudyLanguageDao.class);
+            List<Long> defaultLanguages = studyLanguageDao.getStudyLanguageSql().selectDefaultLanguageCodeId(configuration.getStudyId());
+            if (org.apache.commons.collections4.CollectionUtils.isNotEmpty(defaultLanguages)) {
```

Comment:
Ah, I see we have already imported `org.apache.commons.collections.CollectionUtils`. Maybe we should update code to use the `collections4` one? We can also just drop the usage of `isNotEmpty()` and simply check `!defaultLanguages.isEmpty()`. JDBI won't return nulls if return type is a collection.